packages/module/src/LogViewer/LogViewer.tsx

```diff
@@ -113,13 +115,21 @@ const LogViewerBase: React.FunctionComponent<LogViewerProps> = memo(
     let resizeTimer = null as any;
 
     useEffect(() => {
+      let observer : ResizeObserver | undefined = undefined;
       if (containerRef && containerRef.current) {
         window.addEventListener('resize', callbackResize);
+        observer = new ResizeObserver((event) => {
```

Comment:
Could you check the event type here @mmuzikar? if `event.type === 'scroll'` then we don't do the `callbackResize`, otherwise we could use `callbackResize`.

Reply:
Do you mean the event in the ResizeObserver? Because there is no type there. Or is this comment meant for the line above?

Reply:
Yes, I mean the event here, sorry I thought this is an HTML DOM event here.

Reply:
@DaoDaoNoCode I took another look at this and realized `Drawer` has an `onResize` callback. I wonder if somehow you could come at it from that angle instead, and somehow trigger the LogViewer to recalculate its layout imperatively when the drawer is resized. It would of course be nice to have a general solution where the LogViewer handles this itself, but perhaps that would be less hacky? I'm not sure how you would trigger it from outside the LogViewer, just a thought.

Reply:
> @DaoDaoNoCode I took another look at this and realized `Drawer` has an `onResize` callback. I wonder if somehow you could come at it from that angle instead, and somehow trigger the LogViewer to recalculate its layout imperatively when the drawer is resized. It would of course be nice to have a general solution where the LogViewer handles this itself, but perhaps that would be less hacky? I'm not sure how you would trigger it from outside the LogViewer, just a thought.

@mturley I tried using `onResize` callback on drawer in this PR: https://github.com/opendatahub-io/odh-dashboard/pull/2184
Discussion : https://github.com/opendatahub-io/odh-dashboard/pull/2184#discussion_r1433180331 
But it didn't work as expected.  `onResize` it was not recalcualting the lineHeight. Line height recalculation was happening when the window resizes.

GIF:
[Screencast from 2023-11-20 17-55-51.webm](https://github.com/patternfly/react-log-viewer/assets/99261071/9d873e6f-ee56-4684-9811-6d4a837ff575)

Reply:
I updated my PR with the changes you requested, I am not sure I'm following the conversation as I am no React expert, if you have a better idea on how to fix this issue, I can close this and let you do it the right way :)